src/ethereum.js

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect, useMemo } from "react";
+import { parseUnits } from "./helpers";
+import { ethers } from "ethers";
+import ERC20JSON from "@openzeppelin/contracts/build/contracts/ERC20";
+
+const { hexlify } = ethers.utils;
+
+export function useEthereumAccounts() {
+  const [accounts, setAccounts] = useState();
+  useEffect(async () => {
+    if (window.ethereum) {
+      setAccounts(await ethereum.request({ method: "eth_accounts" }));
+      window.ethereum.on("accountsChanged", setAccounts);
+    }
+  }, []);
+  return accounts;
+}
+
+export async function sendETH({ to, value }) {
+  const signer = new ethers.providers.Web3Provider(window.ethereum).getSigner();
+  const tx = await signer.sendTransaction({
+    to,
+    value,
+  });
+  return tx.wait();
+}
+
+export async function sendTokens({ token, to, value }) {
```

Comment:
Are we sending tokens to that peer-address mentioned earlier?

Reply:
yep! Although since Ethereum is the only peerchain this just handles sending ethereum tokens. Once we add other chains we might change this to sendETHTokens or something like that.